src/views/Login.vue

```diff
@@ -78,6 +96,12 @@
       load(): void {
         this.isLoading = true
       },
+      BaseUrl(): void {
+        window.open(this.$store.getters.getOdooUrl() + "/web/reset_password")
+      },
+      createAccountRedirect(): void {
+        window.open(this.$store.getters.getOdooUrl() + "/web/signup")
+      },
       async submit(): Promise<void> {
```

Comment:
@SeddikKadi sorry to be nitpicking, but since `@click="whatever"` are actions, I would like to keep the consistency and name **both** methods as actions. The `createAccountRedirect` method goes in the right direction, but is not fully meaningful:
- `BaseUrl` => `openResetPasswordUrl`
- `createAccountRedirect` => `openSignupUrl`

Does that make sense to you ?

Reply:
@SeddikKadi This also needs to be adressed...